sympy/tensor/array/array_comprehension.py

```diff
@@ -48,33 +47,44 @@ def __new__(cls, function, *symbols, **assumptions):
 
     @property
     def function(self):
-        """
-        Return the function applied across limits
+        """Use for obtaining the function applied across limits
```

Comment:
the words `Use for` do not give any info. Simpler sentences better. If a user uses the function, it's pretty obvious he's using it.

Reply:
Okay, I will remove Used for and also the `Returns` part as both `Summary` and `Returns` are quite similar and will lead to redundancy.